pkg/controller/plan/adapter/openstack/builder.go

```diff
@@ -373,12 +373,14 @@ func (r *Builder) mapVideo(vm *model.Workload, object *cnv.VirtualMachineSpec) {
 	if imageVideoModel, ok := vm.Image.Properties[VideoModel]; ok {
 		videoModel = imageVideoModel.(string)
 	}
+	var autoAttachGraphicsDevice bool
 	switch videoModel {
 	case VideoNONE:
+		autoAttachGraphicsDevice = false
 	default:
-		autoAttachGraphicsDevice := false
-		object.Template.Spec.Domain.Devices.AutoattachGraphicsDevice = &autoAttachGraphicsDevice
+		autoAttachGraphicsDevice = true
```

Comment:
wouldn't it be more expressive to replace this with `autoAttachGraphicsDevice := videoModel != VideoNONE`?